config/initializers/sentry.rb

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
 
-Raven.configure do |config|
+Sentry.init do |config|
   config.dsn = Convection.config.sentry_dsn if Convection.config.sentry_dsn
-  config.processors -= [Raven::Processor::PostData]
```

Comment:
If Im reading the migration docs right, I think this is OK to remove as these processors are no longer used in the sdk and there's default data scrubbing behaviors

https://docs.sentry.io/platforms/ruby/migration/#removed-processors